examples/petsc/bps.c

```diff
@@ -397,39 +457,94 @@ int main(int argc, char **argv) {
     CeedVectorSetArray(xcoord, CEED_MEM_HOST, CEED_OWN_POINTER, xloc);
   }
 
-  // Create the Q-function that builds the diff operator (i.e. computes its
+  // Create the Q-function that builds the operator (i.e. computes its
   // quadrature data) and set its context data.
-  CeedQFunctionCreateInterior(ceed, 1,
-                              Setup, __FILE__ ":Setup", &qf_setup);
+  switch (bpChoice) {
+    case CEED_BP1:
+      CeedQFunctionCreateInterior(ceed, 1,
+                                  SetupMass, __FILE__ ":SetupMass", &qf_setup);
```

Comment:
These don't actually come from this file, but rather `bp1.h`, right?

Reply:
I created `__DIR__` and `PATH(headderfile.h)` to handle this. If this is reasonable, I'll transfer it to the other files, especially in #234.

Reply:
Actually, it seems that OCCA likes what I did less. Investigating.

Reply:
Fixed. There are a few instances of this already in the test suite, but we've gotten lucky because `foo.c` and `foo.h` shared the same filename.